grammars/raku.tmLanguage.json

```diff
@@ -2779,7 +2779,7 @@
               "name": "variable.other.identifier.raku"
             }
           },
-          "match": "(?x)\n(\\$|@|%|&)\n(\\.|\\*|:|!|\\^|~|=|\\?)?\n(\n    (?:[\\p{Alpha}_])           # Must start with Alpha or underscore\n    (?:\n       [\\p{Digit}\\p{Alpha}_]  # have alphanum/underscore, or a ' or -\n    |                           # followed by an Alpha or underscore\n       [\\-'] [\\p{Alpha}_]\n    )*\n)",
+          "match": "(?x)\n(\\$|@|%|&)\n(\\.|\\*|:|!|\\^|~|=|\\?)?\n(\n    (?:[\\pL\\pM_])           # Must start with Alpha or underscore\n    (?:\n       [\\\\p{Digit}\\pL\\pM_]  # have alphanum/underscore, or a ' or -\n    |                           # followed by an Alpha or underscore\n       [\\-'] [\\pL\\pM_]\n    )*\n)",
```

Comment:
Is `\\\\p{Digit}` a mistake?

Reply:
Yes, I think that was a mistake but it had been fixed by the time you saw the PR, no? Github even says that the file you commented on is an outdated view.

Reply:
> it had been fixed by the time you saw the PR, no?

I don't know. I just got an opportunity to do stuff online, pulled up this PR last night, did a review, and it was the only thing I spotted that didn't look right.

I didn't notice anything about anything being outdated. I do see an "outdated" tag now though. Maybe I missed it? I've only tried to use the PR review feature a few times before, and I don't think I've reviewed an outdated file before. (I'm a bit surprised they don't disallow reviewing of an outdated file, or at least make the warning more prominent. Then again, now I've made that mistake I will be looking out for signs of it in the future, so maybe it doesn't need to be more prominent.)

Anyhow, suffice to say I've now reviewed this PR and it "looks good" to me. (That said, I've not tested it, and won't be testing it.)

Reply:
Weirdly, after writing my prior comment, and then clicking "View reviewed changes", I arrived at a new page (this one) that shows my review comment above, and your reply, and then the comment I just wrote, but does *not* show the "outdated" tag against the file name.

And I'm now no longer in review mode (there's a green button below this comment that says "Start a review") and am just about to click the grey "Add single comment" button that's next to the green "Start a review" button. I've no idea if this comment will land underneath the ones above that I just mentioned but let's see...